apt-hook/hook.cc

```diff
@@ -34,10 +34,21 @@
 #include <libintl.h>
 #include <locale.h>
 
-#define CONTRACT_EXPIRY_STATUS_MESSAGE_TEMPLATE_PATH              "/var/lib/ubuntu-advantage/messages/contract-expiry-status.tmpl"
-#define CONTRACT_EXPIRY_STATUS_MESSAGE_STATIC_PATH                "/var/lib/ubuntu-advantage/messages/contract-expiry-status"
-#define CONTRACT_EXPIRY_STATUS_APT_MESSAGE_TEMPLATE_PATH          "/var/lib/ubuntu-advantage/messages/contract-expiry-status-apt.tmpl"
-#define CONTRACT_EXPIRY_STATUS_APT_MESSAGE_STATIC_PATH            "/var/lib/ubuntu-advantage/messages/contract-expiry-status-apt"
+#define MOTD_ESM_SERVICE_STATUS_MESSAGE_STATIC_PATH      "/var/lib/ubuntu-advantage/messages/motd-esm-service-status"
```

Comment:
We not fracture the messages into fragments, and apt-hook renders the appropriate NO_PKGS vs PKGS template based on what esm service pkg counts are. Ultimately after rendering partial messages process_all_templates concatonates the fragments into APT_PRE_INVOKE_MESSAGE_STATIC_PATH and MOTD_ESM_SERVICE_STATUS_MESSAGE_STATIC_PATH